### PR TITLE
feat(windows): add text editor to the support makefile

### PR DESCRIPTION
**Pull request by rc-swag**
- Add text editor to the `windows/src/support/Makefile` 
- Fixed the TextEditor Makefile to copy the debug file to the debug folder.

# User Testing

# TEST_DOWNLOAD_USE_TXT_EDITOR

* Under the `Test Artifacts` heading on this PR click the `Text Editor (32 bit)`
* Expected result: it should download. 
* Under the `Test Artifacts` heading on this PR click the `Text Editor (64 bit)`
* Expected result: it should also download.
* You should be able to run either of these once downloaded.

### CODE REVIEW

**Review by mcdurdin — APPROVED** [review]
LGTM, we should review when it has finished building to ensure that the artifact is visible. Also, be good to add it to status.keyman.com for the test bot to link to texteditor automatically in the User Testing Results comment.

**Review by ermshiperete — APPROVED** [review]

**Comment by mcdurdin** [comment]
> we should review when it has finished building to ensure that the artifact is visible

image

Looks good.

> Also, be good to add it to status.keyman.com for the test bot to link to texteditor automatically in the User Testing Results comment.

* https://github.com/keymanapp/status.keyman.com/issues/352

**Comment by mcdurdin** [comment]
And now ...

image

**Comment by rc-swag** [comment]
> ... has finished building to ensure that the artifact is visible. Also, be good to add it to status.keyman.com for the test bot to link to texteditor automatically in the User Testing Results 

This is also why I had a TODO: on the User Testing because I wanted to see how it would end up looking. Should be simple for the test team to use

**Comment by bharanidharanj** [comment]
# Test Results

* **TEST_DOWNLOAD_USE_TXT_EDITOR (PASSED):** Tested this PR as per the instructions and here is my observation: 1. Able to download Text Editor (32-bit ). exe file without having any problem. 2. Able to download Text Editor (64-bit) .exe file without having any problem. 3. Clicking on the .exe file, it opens the Text Edit app. Seems to be working as expected. 

..downloaded 32-bit and 64-bit Text Editor .exe files





..Able to open Text Editor

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 17.0.108-alpha](https://keyman.com/downloads/releases/alpha/17.0.108)

**Pull request merged**
